### PR TITLE
PHP 'Hello World' Tutorial added

### PR DESCRIPTION
**Pull request by eddiejaoude**
PHP is such a popular language for Dynamic Websites and Web Applications. 

I have created a *Lesson 1*. I will ping this link to some friends and colleagues for review.

### CODE REVIEW

File: php/lesson1/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+```bash
+$ php -S 0.0.0.0:8080
+```
+
+1. You are already familiar with the `php` command from before
+2. `-S` means built-in WebServer
+3. `0.0.0.0` is the IP that the WebServer should listen on. By using `0` it will listen on everything - fine for development
+4. `8080` is the port to listen on - fine for development but in production the default port is `80` and therefore not required when accessing a URL
+
+Lets see the script output in the browser, in your browser (ideally **Firefox** or **Chrome**) navigate to ` http://localhost:8090/` and you should see:
```
Comment (randompixel) [inline php/lesson1/tutorial.md]: You say listen on 8080, but then direct them to 8090. Is this right?

Comment (eddiejaoude) [inline php/lesson1/tutorial.md]: Well noticed! I will update it. Thanks

**Comment by eddiejaoude** [comment]
> One other thing to consider? Use repository name that keeps codebar's URI clear. See mine for example! ^-^ 

@kaichanvong what do you mean?

**Comment by kaichanvong** [comment]
@eddiejaoude – exactly what was said. And sneaky sneaky moving the chat here! But OK hehe ^_^

**Comment by eddiejaoude** [comment]
> exactly what was said

I didn't understand 'what was 'said', hence why I was asking.

> And sneaky sneaky moving the chat here

Moving the chat? Sorry you lost me...I had multiple tabs open and just used one that was in the PR. What is the difference?

**Comment by kaichanvong** [comment]
Oh. Okay. It's a fun problem to try figure out. Or chat to me in person, or someone else about it at a Codebar session! hf gl!

`^hf gl!^Have fun, good luck (not mean't sarcastically as some people may think)!`

**Comment by eddiejaoude** [comment]
> It's a fun problem to try figure out

Which *problem* are you referring to? I didn't realise there was one :smile: 

Thanks for your feedback though, well the ones I understood :stuck_out_tongue_winking_eye:

**Comment by deniseyu** [comment]
Hey @eddiejaoude thanks for your work on this. minor nitpick -- could you include a link to install in case it's not on the OS? Would also be useful to prompt the user to run `php -v` with the expected output to sanity-check that php is successfully installed.

**Comment by eddiejaoude** [comment]
@deniseyu good point. I will update it shortly.

**Comment by eddiejaoude** [comment]
### TODO

* [ ] Link on how to install **PHP**
* [ ] Create another tutorial/PR for Cloud9IDE (c9ide)
   * [ ] login in with GitHub account
   * [ ] create PHP environment
   * [ ]  run CLI command to see output
   * [ ] run apache & browser to see output

**Comment by eddiejaoude** [comment]
@deniseyu I have added an installation link to the official documentation and the check command to make sure it is installed correctly.

**Comment by deniseyu** [comment]
:+1:

**Pull request merged**